test/utilities/miscellaneous.ts

```diff
@@ -43,3 +42,53 @@ export function currentOsUserName(): string {
 export function transformedUserName(): string {
   return currentOsUserName().replace('.', '_');
 }
+
+export async function findElementByText(
+  type: string,
+  attribute: string,
+  labelText: string
+): Promise<WebdriverIO.Element> {
+  let element = await $(`${type}[${attribute}="${labelText}"]`);
+  return element!;
+}
```

Comment:
I like this helper function!  Can you add a comment on top of it to say what it does?

Reply:
Same for the getTextEditor() helper function at the bottom of this file.

Reply:
done and pushed 🙂

Reply:
Thank you!